python/lsst/meas/extensions/scarlet/deblend.py

```diff
@@ -222,36 +222,22 @@ class ScarletDeblendConfig(pexConfig.Config):
                                          "iterations to exit fitter"))
 
     # Blend Configuration options
-    recenterPeriod = pexConfig.Field(dtype=int, default=5,
-                                     doc=("Number of iterations between recentering"))
-    exactLipschitz = pexConfig.Field(dtype=bool, default=True,
-                                     doc=("Calculate exact Lipschitz constant in every step"
-                                          "(True) or only calculate the approximate"
-                                          "Lipschitz constant with significant changes in A,S"
-                                          "(False)"))
```

Comment:
These seem like they should be on their own commit, I dont see how it relates to the edge work

Reply:
Agreed. I came across obsolete parameters when I was adding the edge ones and rather than forget to remove them I deleted them right away. But it seems like a small enough change to not warrant the time that it would take to pick lines for a separate commit.